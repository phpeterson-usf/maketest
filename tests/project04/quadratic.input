-h $(repo_dir)/quadratic.hex $(TESTS_DIR)/quadratic_s.s
