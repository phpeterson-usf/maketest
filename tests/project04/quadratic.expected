v2.0 raw
e00c0090
e0010c91
e0020290
e0810002
e0800003
e12fff1e
